mlrun/db/factory.py

```diff
@@ -59,9 +61,15 @@ def create_run_db(self, url="", secrets=None, force_reconnect=False):
             #  The SQLRunDB should always get its session from the FastAPI dependency injection.
             self._run_db = self._rundb_container.run_db(url)
 
+        self._verify_ssl()
         self._run_db.connect(secrets=secrets)
         return self._run_db
 
+    @staticmethod
+    def _verify_ssl() -> None:
```

Comment:
you dont verify ssl here, you disable warnings upon user's preferences.

I think the functions logic can be moved to `read_env` (I know @quaark had some objections about it but this is the current place to put config -> logic functionality) and it will fix the scenario where user change the config during runtime

https://github.com/mlrun/mlrun/blob/d444546f8a3f02d47987c0188b607b86c03cabb6/mlrun/config.py#L1429-L1434